src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -140,7 +140,7 @@ class SignInModal extends React.Component {
           </div>
           <div className="columns usa-width-one-half medium-6">
             <div className="explanation-content">
-              <div className="hide-for-small usa-font-lead">
+              <div className="vads-u-display--none medium-screen:vads-u-display--block usa-font-lead">
```

Comment:
There's some weirdness going on with this component. It's mixing together USWDS's grid, Foundation's grid, and our breakpoints and Foundation's breakpoints. As it stands, this change makes this text disappear before it should. It looks like it's supposed to be hidden when the modal shifts to a single column, but this makes it disappear before then. And changing it to `small-screen` doesn't help, because then it doesn't disappear until really narrow widths.

I played around with it a little bit and I think we can fix it by:

1. Leave the classes on this element alone
2. Remove the `columns` and `medium-6` classes on the two divs in this component, along with the `row` class on the div that surrounds them.
3. Remove the right margin on `signin-actions-container`
4. Add a left margin of 2em on `explanation-content`, for medium screen sizes and up.

Reply:
@jbalboni I made the changes you requested. I added the margin left change for a `$medium-large-screen` because it didn't look right on mobile.